information-system/provenance/overview/what-are-entities.md

```diff
@@ -8,14 +8,58 @@ parent: Provenance Overview
 ---
 # What are entities?
 
-In the RRAP M&DS Information System (IS), entities are things that are registered in the M&DS IS Registry. Registering entities is fundamental to establishing stable provenance. 
+## 1. Overview
 
-Each registered item (entity) is given a unique and persistent identifier (ID). These IDs provide a mechanism for 2 things:
+In the RRAP M&DS Information System (IS), entities are things that are registered in the M&DS IS Registry. Registration of entities is fundamental to establishing stable provenance. It establishes a record for each entity used or generated in producing or delivering an artifact in a provenance record. 
+
+In the RRAP M&DS IS, each registered item (entity) is given a unique and persistent identifier (ID). These IDs provide a mechanism for 2 things:
 1. It allows information to be attached to each entity in a defined way
 2. Entities are able to link to other entities (i.e. object references rather than just text descriptions)
 
-These registered things can then be referred to in provenance records, e.g. a provenance record about a model run is able to link to defined descriptions of the model that was used, who ran the model, a list of input data, and the outputs/results. The ability to link to entities reduced manual error and duplication of descriptions, as they can be reused and simply referred to. Registering provenance records with the entity references provides a way to create provenance graphs. These provenance graphs can then be queried. 
+In addition to this, the persistent IDs for each entity in the RRAP M&DS IS is able to be hyperlinked and looked up in the IS via the web.
+
+These registered entities can then referenced in provenance records, e.g. a provenance record about a model run references a computational model that was used, the agent or person who ran the model, a list of input datasets, and the output dataset/results. The ability to reference entities reduces manual error and duplication of descriptions, as they can be reused and simply referred to. 
+
+Registering provenance records with references to the registered entity provides the IS with the ability to create provenance graphs. Users of the IS are then able to explore and query these provenance graphs. 
```

Comment:
'with references' --> 'using cross-references' ?

Reply:
Is 'provenance graph' defined anywhere?

Reply:
It isn't, but i've added one in with a diagram